modules/compose.mjs

```diff
@@ -267,7 +278,7 @@ class ReplyWithHeader {
         let rwhHeaders = '<div id="rwhHeaders"';
         if (await rwhSettings.isHeaderHtmlPrefixLine()) {
             let borderColor = await rwhSettings.getHeaderHtmlPrefixLineColor();
-            rwhHeaders += ` style="border:none;border-top:solid ${borderColor} 1.0pt;padding:3.0pt 0cm 0cm 0cm"`
+            rwhHeaders += ` style="border-top:solid ${borderColor} 1px;padding-top:3px"`
```

Comment:
@ygoe I obtained this `style` code from the Outlook mail client. It works on all email clients for some reason; unit `px` does not always work. So far, my understanding is that HTML CSS style(s) on email content interferes with unit `px` a lot. 

Also, it is good to reset and set the style you need when overriding eg.: `border:none;border-top:solid...`

Reply:
I have reverted all style optimisations here. Although I've never experienced issues with the px unit in e-mails. Actually I see it a lot in extensively formatted e-mails I get, including for example Stackoverflow notifications.